grammar.js

```diff
@@ -1006,6 +1003,8 @@ const rules = {
 
   xhp_class_identifier: $ => /:[a-zA-Z_][a-zA-Z0-9_]*([-:][a-zA-Z0-9_]+)*/,
 
+  _xhp_identifiers: $ => choice($.xhp_identifier, $.xhp_class_identifier),
```

Comment:
Let's make `_xhp_identifiers` singular. A bit of a nit, but I'd reserve plural for a rule that defines a collection like `arguments` or `parameters`. This rule really stands for a single item.

Let's also add `_xhp_identifier` to the inline rules. We don't really need the syntax tree node wrapping the two rules at runtime.